run.py

```diff
@@ -165,5 +192,5 @@ def transform_and_save(data:dict):
         print(saved_data)
 # End of transform_and_save
 
-transform_and_save(data_selected_processed)
+#transform_and_save(data_selected_processed)
```

Comment:
It looks like this whole file is used for testing purposes.  If that's the case, this should all get moved into pytest so that it runs with our testing suite.  We can also use a library like faker and cookyCutter to mock out data objects to use for testing.  There are the rock object libraries as well.  That will allow you to test our functions without hard coding the objects.

Reply:
Will took sometime to incorporate properly into pytest, rightnow i am using it for manual testing

Reply:
Understandable, I added a task in Zenhub for us to polish up our API testing suite.  Should probably be one of the next things we do with the API so we can continue to refactor with confidence going forward.